### ISSUE TITLE
Components' css is not loaded in static storybook

### ISSUE THREAD

**Issue by HelloDeadline** [storybooks/vue-cli-plugin-storybook#27]
I created new vue app using Vue CLI 3. Added vue-cli-plugin-storybook and downgraded storybook dependencies (see the [issue](https://github.com/storybooks/vue-cli-plugin-storybook/issues/22#issuecomment-419990177))

Serve works fine. But when I build static storybook and try to serve it with simple http server, there is an error with css:

```
Uncaught SyntaxError: Unexpected token {      // iframe.44ba0317.css:1
```

I guess this might be because `iframe.44ba0317.css` is injected as script tag in `storybook-static/iframe.html`:

```html
<script src="css/iframe.44ba0317.css" defer></script>
```

Example repo:
https://github.com/HelloDeadline/storybook-vue-css-bug

Steps to reproduce:

```
yarn install
yarn build:storybook
yarn serve:build:story
```

**Comment by ram33** [comment]
I am facing the same issue. Fixed it manually.

**Comment by pksunkara** [comment]
@ram33 How did you fix it manually?

**Comment by ram33** [comment]
@pksunkara by changing them from scripts to links

before
![image](https://user-images.githubusercontent.com/12947297/46073067-934d1d00-c1a1-11e8-8a9e-fb1e48d5831e.png)

after
![image](https://user-images.githubusercontent.com/12947297/46073158-c394bb80-c1a1-11e8-8bf8-ae41d6d61d80.png)

**Comment by HelloDeadline** [comment]
Unfortunately I don't have enough time and experience to discover the issue and contribute to the project, but I wrote small script that helps us fix this problem in generated file automatically after the build:

```
const fs = require('fs')
const path = require('path')

const pathToHtml = path.resolve(__dirname, '../storybook-static/iframe.html');
const getScriptTag = (path) => new RegExp(`\\<script(.*)?${path.replace(/\./g, '\\.')}\"(.*)?\\<\\\\script\\>`, 'g')
const getLinkTag = (path) => `<link rel="stylesheet" href="${path}">`

const replaceContents = (contents, matchR) => {
    const match = contents.match(matchR)

    if (!match) {
        return contents
    }
    
    let correctContents = contents;
    
    match.forEach(m => {
        const source = m.match(/css(.*)?\.css/)
        if (!source) {
            return
        }

        correctContents = correctContents.replace(m, getLinkTag(source[0]));
    })

    return correctContents
}

fs.readFile(pathToHtml, {encoding: 'utf8'}, function(error, contents) {
    if (error) {
        console.error(error)
        return
    }

    const correctContents = replaceContents(contents, /<script(.*)?\.css(.*)?script>/g)
    fs.writeFileSync(pathToHtml, correctContents, {encoding: 'utf8'})
})
```

So in `package.json` we have: `"build:storybook": "vue-cli-service build:storybook -s ./public -c config/storybook && node ./build-scripts/fix-static-storybook.js",`

**Comment by ram33** [comment]
@HelloDeadline Its working like a charm. Thanks for the workaround.

**Comment by HelloDeadline** [comment]
> @HelloDeadline Its working like a charm. Thanks for the workaround.

Glad it helped :)

**Comment by rndmerle** [comment]
Hi.
Facing the same issue.
Thanks for the workaround but anyone got the time to work on a fix hopefully?

**Comment by anhmv** [comment]
This issue was fixed in v0.5.0
